### ISSUE TITLE
Some extra work is required to get ahash to work with rayon

### ISSUE THREAD

**Issue by kimono-koans** [tkaitchuck/aHash#120]
Not a bug, this is just FYI.  Given how common the use of `rayon` is, this is something you may wish to provide documentation for, or even implementations of the `rayon` traits.  See the linked `rayon` issue: https://github.com/rayon-rs/rayon/issues/953

As I stated in that issue, everything works fine once I knew what to do, so just a small blurb for the user might be best.

Appreciate all your hard work.  Thank you.

**Comment by dimfeld** [comment]
PR #105 added the `ahash::HashMap` and `ahash::HashSet` type aliases, which you can use to get rayon to work without having to define your own.

**Comment by tkaitchuck** [comment]
Yes. These aliases are the prefered solution. I've added a sentence mentioning Rayon in the documentation.
